components-2/readme.md

```diff
@@ -0,0 +1,166 @@
+
+### Each team, choose a driver
+
+1. Driver, ask everyone in your team to confirm you are in the right working branch.
+1. Driver, from this branch, create and checkout a new branch for your team named like this: `feature/component`, where _cohort_ is your cohort name and _component_ is the component you're building, for example: `feature/header`
+
+```bash
+git switch --create feature/cohort-component
+```
+
+Now you're all set up to make changes to together.html.
+
+<!--{{</note>}}-->
+
+<!--{{<note type="activity" title="Find your docs 10m">}}-->
+
+## Find your docs
+
+Set a whole class [timer for 10 minutes](https://www.google.com/search?q=10+minute+time).
+
+Deployed from this repo is a [mini design system](https://cyf-workshops.netlify.app/components-2). This design system has several documented components, and some also have javascript files.
+
+Your mission is to find the docs for each component with an associated script file, and read them. Discuss your findings. Look at the pages on the website with Devtools. Compare the Inspect view with the source code you can see in the files. How are they different? How are HTML and the DOM different?
+
+<!--{{</note>}}-->
+
+### Together separately
+
+Set a whole class [timer for 30 minutes](https://www.google.com/search?q=30+minute+time).
+
+<!--{{<note type="activity" title="Parallel Development 30m">}}-->
+
+Now **build** your component. Elect a new driver.
+
+1. Find [together.html](together.html) in the /components-2 directory. Your mission is to complete this page.
+1. In the correct portion of the page, make your changes to build _your feature_ on together.html.
+1. Driver, commit your changes to your feature branch.
+1. Open a pull request from your branch to the COHORT branch, with your changes. Ask for help!
+1. Remember, docs will help you too.
+1. Do the simplest thing that could possibly work to begin with. You can always update your PR with more changes.
+
+```mermaid
+gitGraph
+    commit id: "start"
+    branch feature/component
+    commit id: "import component function"
+    commit id: "import data"
+    commit id: "lighthouse revisions"
+    checkout COHORT
+    merge feature/component
+```
+
+<!--{{</note>}}-->
+
+### Review and merge
+
+Now we're going to review each other's work and merge to our cohort branch.
+
+<!--{{<note type="warning" title="Your working branch is your cohort name">}}-->
+
+Your COHORT branch is your cohort name. You're working in your COHORT branch and NOT in main. Open your pull request to your COHORT branch. Don't open PRs to main.
+
+<!--{{</note>}}-->
+
+Set a whole class [timer for 30 minutes](https://www.google.com/search?q=30+minute+time).
+
+<!--{{<note type="activity" title="Review 30m">}}-->
+
+1. Go to the [pull requests](https://github.com/CodeYourFuture/CYF-Workshops/pulls) for this repo.
+1. Either individually or as a group, review each other's PRs.
+1. Once you're all happy with the PR, merge it.
+1. Did it work? Why/why not? Discuss.
```

Comment:
Do we need to more clearly define what "worked" actually means?

I guess my underlying question is whether trainees will consider "clicking the merge button" as "working". So we might need to put in something about checking the outcome in `together.html`

But I can also see that coming out of the discussion, especially if guided by volunteers.

Reply:
I think we can bear a line prodding them to look at together.html  😉